aboutcode/pipeline/README.md

```diff
@@ -0,0 +1,53 @@
+# `aboutcode.pipeline`
+
+Define and run pipelines.
+
+### Install
+
+```bash
+pip install aboutcode_pipeline
```

Comment:
As per PEP8 https://peps.python.org/pep-0008/#package-and-module-names `aboutcode-pipeline` would be a much more Pythonic package name.
```suggestion
pip install aboutcode-pipeline
```

Reply:
I initially went with aboutcode_pipeline in the README as the filenames of the flot build were `aboutcode_pipeline-0.1.whl`, even though the [project] name was set to "aboutcode.pipeline".
@pombredanne Any input on this?